### PR TITLE
created an action type and an action creator for ADD_TODO_LIST

### PR DESCRIPTION
**Pull request by swkane**
path: actions/index.js
exporting: 
 -  ADD_TODO_LIST (variable/action type)
 -  addTodoList (action creator)

### CODE REVIEW

File: src/actions/index.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+// Action type
+const ADD_TODO_LIST = 'ADD_TODO_LIST';
+
+// Action Creator (a function that creates an action)
+
+// Action Creator w/ Action type
+
+const addTodoList = title => {
```
Comment (declanvea) [inline src/actions/index.js]: Not being exported, please update

Comment (declanvea) [reply src/actions/index.js]: Check errything

**Review by declanvea — CHANGES_REQUESTED** [review]
Ran into issue exporting issue with updates, keep up the good work (but better)!

File: src/actions/index.js (hunk 1 above)
Comment (swkane) [reply src/actions/index.js]: @declanvea fixed these changes

**Review by declanvea — APPROVED** [review]
Great job!

**Pull request merged**